Feat/6373 example gold workload (#197)
Ensono/stacks-azure-data: docs/workloads/azure/data/etl_pipelines/data_processing.md

Reviewer: As a reminder - these updates will also need applying in the main docs repo.

![ADF_SilverPipelinesList.png](../images/ADF_SilverPipelinesList.png)

> In Data Factory a standard data processing pipeline is as simple as this example:

Reviewer: ```suggestion
In Data Factory a data processing pipeline can be as simple as this example:
```
